src/store/reducers.js

```diff
@@ -0,0 +1,55 @@
+const defaultState = {
+  trendingGifs: [],
+  favoriteGifs: [],
+  error: '',
+  loading: false,
+  singleGif: null
+}
+
+const reducers = (state = defaultState, action) => {
```

Comment:
It is discouraged to have a big reducer for the whole app. It is a good practice to split them in their own concern-substate.